internal/providers/kubernetes/provider.go

```diff
@@ -180,6 +181,16 @@ func (p *Provider) getServiceMode(annotations map[string]string) string {
 	return mode
 }
 
+func (p *Provider) getScheme(annotations map[string]string) string {
```

Comment:
since this doesn't actually use the provider at all, did we want to have this method moved to the `base` package so that we can use it with the SMI provider?